pandas/_libs/lib.pyx

```diff
@@ -1110,6 +1110,7 @@ _TYPE_MAP = {
     "complex128": "complex",
     "c": "complex",
     "string": "string",
+    "arrow_string": "string",
```

Comment:
PR as draft. This will be resolved in a follow-up to #40679. https://github.com/pandas-dev/pandas/pull/40679#issuecomment-809642594

the inference is needed so that the str accessor works.